lib/rubocop/target_finder.rb

```diff
@@ -94,7 +94,9 @@ def find_files(base_dir, flags)
                   # on the top level, which would otherwise not be found.
                   wanted_toplevel_dirs.unshift("#{base_dir}/*")
                 end
-      Dir.glob(pattern, flags).select { |path| FileTest.file?(path) }
+      Dir.glob(pattern, flags)
+         .select { |path| FileTest.file?(path) }
+         .map { |relative_path| File.expand_path(relative_path) }
```

Comment:
Nitpicky, but if you are worried about performance, using `each_with_object` is better than chaining `select...map`.
```suggestion
Dir.glob(pattern, flags).each_with_object([]) do |path, files|
  next unless FileTest.file?(path)
  files << File.expand_path(relative_path)
end
```

... I don't think github's suggested change feature will work here since my suggestion spans multiple lines.